ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/EVMToolTracer.java

```diff
@@ -34,30 +32,42 @@
 import org.apache.tuweni.bytes.Bytes;
 import org.apache.tuweni.units.bigints.UInt256;
 
-class EVMToolTracer implements OperationTracer {
+public class EVMToolTracer implements OperationTracer {
```

Comment:
Since it's been moved out of EVM tool perhaps a different name, `StandardJsonTracer`?

Reply:
Done